React/Fabric/Mounting/RCTComponentViewFactory.mm

```diff
@@ -57,8 +57,7 @@ + (RCTComponentViewFactory *)standardComponentViewFactory
   [componentViewFactory registerComponentViewClass:[RCTParagraphComponentView class]];
   [componentViewFactory registerComponentViewClass:[RCTTextInputComponentView class]];
 
-  Class<RCTComponentViewProtocol> imageClass = RCTComponentViewClassWithName("Image");
```

Comment:
Hello,

this needs to stay as is. Is allows us to have multiple implementations of Image component view and change them based on the app.

Reply:
I'll revert this on the internal diff. Thanks Sam!

Reply:
Would adding the image component to the OSS `RCTFabricComponentsPlugins.mm` break anything internally at FB?

Reply:
Hey @empyrical,

sorry for the slow reply.

"Image" is already in `RCTFabricComponentsPlugins.mm`. The reason why it is here, and other components above, is for preemptive view allocation. 

Please let me know if you have any other questions.